### DISCUSSION TITLE
Are there any open bounties?

### DISCUSSION THREAD

**Discussion by Kreijstal** [ruffle-rs/ruffle#10686] [General] [answered]
Not that I will contribute, but I don't see them advertised anywhere..
sorry for being nosy.

**Comment by dginovker** [comment, marked as answer]
There's this one https://app.bountysource.com/issues/110151502-champion-archer-bow-not-working-after-first-level

**Reply by dginovker** [reply]
There's also this one: https://app.bountysource.com/issues/95531207-text-not-showing-in-multiple-choice-answers
